app/services/analytics.rb

```diff
@@ -120,7 +122,6 @@ def browser_attributes
   end
 
   # rubocop:disable Layout/LineLength
-  ACCOUNT_RESET = 'Account Reset'
```

Comment:
I picked this one because it was first on the list. It turned out to be super verbose to document because it is essentially 5 different events (it has its own `event` key 🤦 )

I will take some time and try migrating other events to see how much typing they result in

Reply:
Maybe this is a good thing as it uncovers a smell where this should have separate event names for those events?

Reply:
Created a ticket to track splitting it up: https://cm-jira.usa.gov/browse/LG-5910

Reply:
Does splitting it up mean splitting up each event (like this one would be for Account Reset) into a separate document, or does this happen currently? I think with the number of events we have, the documentation can get rather large.

Reply:
Not 100% sure what you're asking? The goal is to have a lot of documentation, so we can better explain what the events in our logs mean, so when we query events, we query them confidently.

But yes, this `analytics_events.rb` file would be expected to get pretty big

Reply:
So the file analytics_events.rb will have all of the events going forward not just account reset such as all the events in analytics.rb?

Reply:
Yeah next steps would be:
1. communicating to the rest of the team that new events should go to `analytics_events.rb`
2. Writing tickets to help migrate all the old events